### DISCUSSION TITLE
[Bug]: multiple file selection implementation issue

### DISCUSSION THREAD

**Discussion by erictioh** [mikekazakov/nimble-commander#318] [Q&A] [answered]
### Confirmation

- [X] I have read the contributing guidelines

### Version of Nimble Commander

MacOS13

### Distribution Type

Downloaded directly from the website (including Brew)

### Expected Result

Problem 1:
Hold down shift / cmd can select some files, noticed that those files turn red font color. However no easy way to deselect those files, i find very long yes i can click back those files 1 by 1 to deselect. I wish the implementation work like finder which click anywhere else can deselect all file, or esc key to deselect.

Problem 2:
select 1 file, the file became red font. Then double click some other file on the same folder, notice that it open only the selected file not the file you double click. I hope you don't say its working as design, it very non intuitive. 

### Actual Result

See expected result

### Steps to reproduce

see expected result

**Comment by mikekazakov** [comment]
Not a **bug**, please refrain from using Issues to ask **questions** or write **ideas**.
I'll convert it to a discussion to answer there.

**Reply by mikekazakov** [reply, marked as answer]
1. Simply place the cursor on any selected file. Then use the arrow keys while holding the Shift modifier to de-deselect instead of select. As well there's an Alt+Cmd+A hotkey that removes the existing selection.
2. I agree that the existing behaviour is not intuitive, need to think what can be done instead.

**Reply by erictioh** [reply]
for no1,  my use case is when i selected few files, then i may forget i have selected, then oftenly came back to the tab and when try to open 1 file all those selected file open up, sometimes i selected 40-50  file it gave me alot of troubles coz it open up 50 windows for those files, then i have to busy closing them.

**Reply by mikekazakov** [reply]
NC writes at the bottom of the panel that there are selected items - you can look for that.
